src/cli/admin/admin-federation-export.ts

```diff
@@ -7,86 +7,90 @@ import {
   exportAdminFederationProviderToFile,
 } from '../../ops/cloud/AdminFederationOps';
 import { printMessage, verboseMessage } from '../../utils/Console';
-import { FrodoCommand } from '../FrodoCommand';
 
-const program = new FrodoCommand('frodo admin federation export', ['realm']);
-
-program
-  .description('Export admin federation providers.')
-  .addOption(
-    new Option(
-      '-i, --idp-id <idp-id>',
-      'Id/name of a provider. If specified, -a and -A are ignored.'
+export const addFederationExport = (command: Command) => {
+  command
+    .command('export')
```

Comment:
Note: the `export` keyword has moved into this file, which makes sense as this file is defining the export command.